tests/test_telemetry.py

```diff
@@ -169,15 +169,13 @@ def test_app_started_sent_exactly_once(self):
     @bug(library="python", reason="app-started not sent first")
     @flaky(library="nodejs", reason="APPSEC-10465")
     def test_app_started_is_first_message(self):
-        """Request type app-started is the first telemetry message or in the first batch"""
+        """Request type app-started is the first telemetry message of the first message of the message batch"""
         telemetry_data = list(interfaces.library.get_telemetry_data(flatten_message_batches=False))
```

Comment:
if you use `flatten_message_batches`, it will transparently handle this use case